repository.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Stale testing file?

Reply:
Sorry, I made a branch for this PR and to host a repository which obviously should be two separate concerns. Understood. Thanks for the review :)